src/science/radiation/cbl_init_radiation.F90

```diff
@@ -378,9 +378,8 @@ SUBROUTINE BeamFraction( RadFbeam, mp, nrb, Cpi,Ccoszen_tols_huge, metDoy,     &
 
 
 ! Define beam fraction, fbeam:
-! #355 beam fraction defined using total SW_down; applies to VIS&NIR equally
-RadFbeam(:,1) = spitter(mp, cpi, metDoy, coszen, SW_down(:,1)+SW_down(:,2))
-RadFbeam(:,2) = RadFbeam(:,1)
+RadFbeam(:,1) = spitter(mp, cpi, metDoy, coszen, SW_down(:,1))
+RadfBeam(:,2) = spitter(mp, cpi, metDoy, coszen, SW_down(:,2))
```

Comment:
That's removing the bug fix from #355 , big no! The bug fix will go into ESM1.6 as a difference compared to ESM1.5

To remove/modify to use the formulation in main (red lines here)

Reply:
yeah this was done somewhere else as well recently. change snuck in from AM3 where spitter is never used anyway and so we didnt have the change in place. A good reason to have the same version is every app